### PR TITLE
Add health check endpoint

### PR DESCRIPTION
**Pull request by nskoufis**
This adds a simple ping health check at /healthz

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -33,6 +34,9 @@ func main() {
 	promProvider := prometheus.NewProvider(promAddress, timeout)
 	cmd.WithExternalMetrics(promProvider)
 
+	server, _ := cmd.Server()
+	server.GenericAPIServer.AddHealthzChecks(healthz.PingHealthz)
```
Comment (pjmichael) [inline main.go]: What does the health check actually do @nskoufis ?

Comment (nskoufis) [reply main.go]: It's really just a liveness check. Just responds with ok, assuming the http server (which also serves the bits and bobs for the api server extension) is up

Comment (nskoufis) [reply main.go]: We could definitely add pinging the prometheus upstream? But does that help?

Comment (pjmichael) [reply main.go]: no was just curious :)

**Pull request merged**